contracts/CrossChainV2.sol

```diff
@@ -0,0 +1,587 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
Is this file necessary, as it is identical with the `CrossChain.sol`

Reply:
removed